source/partials/_footer.slim

```diff
@@ -29,10 +28,11 @@ footer
           | Community
         .u-xSmallMargin
         ul.SitemapTopic-links
-          li = link_to 'Creators Schools', '/community#creatorsschool', class: 'SitemapTopic-link'
-          li = link_to 'RubyConf Portugal', '/community#rubyconfpt', class: 'SitemapTopic-link'
-          li = link_to 'BragaRB', '/community#bragarb', class: 'SitemapTopic-link'
-          li = link_to 'BragaJS', '/community#bragajs', class: 'SitemapTopic-link'
+          li = link_to 'Mirror Conf', 'http://www.mirrorconf.com/', target: :_blank, rel: 'noopener noreferrer', class: 'SitemapTopic-link'
+          li = link_to 'RubyConf Portugal', 'http://rubyconf.pt/', target: :_blank, rel: 'noopener noreferrer', class: 'SitemapTopic-link'
+          li = link_to 'Braga.Design', 'https://www.meetup.com/bragadesign/', target: :_blank, rel: 'noopener noreferrer', class: 'SitemapTopic-link'
+          li = link_to 'Braga.JS', 'https://www.meetup.com/bragajs/', target: :_blank, rel: 'noopener noreferrer', class: 'SitemapTopic-link'
+          li = link_to 'Braga.Product', 'https://www.meetup.com/Braga-Product/', target: :_blank, rel: 'noopener noreferrer', class: 'SitemapTopic-link'
```

Comment:
FYI: 

> For historical reasons, the noreferrer keyword implies the behavior associated with the noopener keyword when present on a hyperlink that creates a new browsing context. That is,
a href="..." rel="noreferrer" target="_blank"
has the same behavior as
a href="..." rel="noreferrer noopener" target="_blank".

https://html.spec.whatwg.org/multipage/semantics.html#link-type-noreferrer